Header header

int32 camAngle
bool centered
